src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -26967,5 +26967,109 @@ public class Outer
             CompileAndVerify(compRelease, expectedOutput: "C1 { I1 = 42 }", verify: Verification.Skipped /* init-only */);
             compRelease.VerifyEmitDiagnostics();
         }
+
+        [Fact]
+        [WorkItem(44571, "https://github.com/dotnet/roslyn/issues/44571")]
+        public void XmlDoc()
+        {
+            var src = @"
+/// <summary>Summary</summary>
+/// <param name=""I1"">Description for I1</param>
+public record C(int I1);
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>Ignored</summary>
+    public static class IsExternalInit
+    {
+    }
+}
+";
+
+            var comp = CreateCompilation(src, parseOptions: TestOptions.RegularWithDocumentationComments);
+
+            var tree = comp.SyntaxTrees[0];
+            var model = comp.GetSemanticModel(tree, ignoreAccessibility: false);
+            var parameter = tree.GetRoot().DescendantNodes().OfType<ParameterSyntax>().First();
+            var i1 = model.GetDeclaredSymbol(parameter);
+
+            comp.VerifyDiagnostics(
+                // (4,21): warning CS1591: Missing XML comment for publicly visible type or member 'C.I1'
```

Comment:
>warning CS1591: Missing XML comment for publicly visible type or member 'C.I1' [](start = 27, length = 78)

Is this warning for generated property? I think we need to suppress it one way or another.

Reply:
Yes, the warning is for the generated property. I'm okay to suppress it